lib/runners/processor/metrics_fileinfo.rb

```diff
@@ -0,0 +1,66 @@
+module Runners
+  class Processor::MetricsFileInfo < Processor
+    Schema = _ = StrongJSON.new do
+      # @type self: SchemaClass
+
+      let :runner_config, Schema::BaseConfig.base
+      let :issue, object(
+        line_of_code: integer?,
+        last_commit_datetime: integer
+      )
+    end
+
+    def analyzer_version
+      @analyzer_version ||= extract_version!("wc")
```

Comment:
[suggestion] `wc` version is inaccurate because `git` also is used. So how about using the placeholder `0.0.0`?

```suggestion
      "0.0.0"
```

Or using the Runners version seems good:

```suggestion
      Runners::VERSION
````

Reply:
fix: 694b4a877ec950708b3df2f72fe765f53cb8d749